web/pom.xml

```diff
@@ -875,10 +800,259 @@
+                <version>${gn.schemas.version}</version>
+              </dependency>
+            </dependencies>
+            <!-- copy schema resources using relative paths -->
+            <executions>
+              <execution>
+                <id>copy-schemas</id>
+                <phase>process-resources</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration>
+                  <includeEmptyDirs>true</includeEmptyDirs>
+                  <overwrite>true</overwrite>
+                  <outputDirectory>${build.webapp.resources}</outputDirectory>
+                  <resources>
+                    <!-- Copy the schema folder to the webapp folder source directory
+                    in order to be loaded by the application when running mvn jetty:run.
+                    -->
+                    <resource>
+                      <directory>${project.basedir}/../schemas/iso19139/src/main/plugin</directory>
+                      <targetPath>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins
+                      </targetPath>
+                    </resource>
+                    <resource>
+                      <directory>${project.basedir}/../schemas/iso19110/src/main/plugin</directory>
+                      <targetPath>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins
+                      </targetPath>
+                    </resource>
+                    <resource>
+                      <directory>${project.basedir}/../schemas/dublin-core/src/main/plugin</directory>
+                      <targetPath>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins
+                      </targetPath>
+                    </resource>
+                    <resource>
+                      <directory>${project.basedir}/../schemas/csw-record/src/main/plugin</directory>
+                      <targetPath>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins
+                      </targetPath>
+                    </resource>
+                    <resource>
+                      <directory>${project.basedir}/../schemas/iso19115-3.2018/src/main/plugin</directory>
+                      <targetPath>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins
+                      </targetPath>
+                    </resource>
+                    <!-- Copy the documentation in the webapp. -->
+                    <resource>
+                      <directory>${project.basedir}/../docs/manuals/target/doc</directory>
+                      <targetPath>${basedir}/src/main/webapp/doc</targetPath>
+                    </resource>
+                  </resources>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- Example: Unpack hnap schema plugin -->
+    <profile>
```

Comment:
I would add this commented as an example or in the developer documentation.

Reply:
Developer docs are managed here in README.md files, as per [this activity](https://github.com/geonetwork/core-geonetwork/pull/4697).